### ISSUE TITLE
Emoji character encoding issue.

### ISSUE THREAD

**Issue by allsoft777** [ical4j/ical4j#68]
Hello. I'm reporting an issue.
While encoding the calendar event, an emoji character of the specific position is broken down.
But, the encoded file is generated successfully.
I guess, it is related with the folding character. SUMMARY, DESCRIPTION and LOCATION have a same issue.

i.e) Fill the field with all the emoji more than 100 length.
The broken emoji is showing every specific position repeatedly.

SUMMARY:?삇?삇?삇?삇?삇?삇?삇?삇?삇?삇?삇?삇?삇?삇?삇?삇?삇?삇?삇?삇?쁼?삇?삇?쁼?쁼?쁼?쁼?삇?삇?삇?삇?삇?삇?
 ??삇?삇?삇?삇?삇?삇

**Comment by benfortuna** [comment]
Based on your description it sounds like it might be an issue with folding long content lines. Do you have some sample code or file that might demonstrate the issue?

**Comment by allsoft777** [comment]
The test code has no special code snippet.
I just used the sample code such as.
http://www.javaxp.com/2012/05/ical4j-example-set-meeting-using.html

Instead of the constant title, i used to the EditText.
String subject = EditText.getText().toString();

The subject edittext have all the emoji characters and the OS is android 6.0.

**Comment by benfortuna** [comment]
Just to confirm, have you logged the output of `EditText.getText().toString()` to verify it is what you expected?

**Comment by allsoft777** [comment]
Yes, the string with the 'EditText.getText().toString()' is normal as i expected.
